src/extensions/segment-integration.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Integrate Sentry with PostHog. This will add a direct link to the person in Sentry, and an $exception event in PostHog
```

Comment:
```suggestion
 * Integrate Segment with PostHog. This will enrich Segment events with custom PostHog data including support for recordings and feature flags.
```

Reply:
and Usage below 😅